### DISCUSSION TITLE
Simple line filters - cannot get it working

### DISCUSSION THREAD

**Discussion by skigoggles** [WinMerge/winmerge#567] [Q&A]
I have an XML file with entries like: 

```
  <type name="AK74">
    <nominal>2</nominal>
    <lifetime>7200</lifetime>
    <restock>1800</restock>
    <min>1</min>
    <quantmin>-1</quantmin>
    <quantmax>-1</quantmax>
    <cost>100</cost>
    <flags count_in_map="1" count_in_hoarder="0" count_in_cargo="0" count_in_player="0" crafted="0" deloot="0" />
    <usage />
    <value user="Tier3" />
    <category name="rifles" />
    <tag name="military" />
  </type>
```

It's a game config file. It may change from release to release.
I want to diff and ignore various lines like:

```
    <nominal>2</nominal>
    <min>1</min>
```
I thought a valid regex would be : 

`
^    <nominal>
`

I had it working, tweaked it and now nothing is working. I've read far too many regex docs that my head hurts. It must be something simple, right?

**Comment by DGH2112** [comment]
Try the following regex: `^\s*<nominal>` assuming the reg ex supported is equivalent to Perl 5 (i.e. `\s*` caters for zero or more spaces).

**Comment by skigoggles** [comment]
thanks! Works on a certain set of files, but on another set it consistently doesn't work. I'm wondering if there is some strange file encoding. The broken ones came from github and I know there is an issue with a few lines where they get validation errors, but nothing is visible in a text editor (I'll have to edit binary and see).
